code/modules/research/techweb/all_nodes.dm

```diff
@@ -11,7 +11,7 @@
 	// Default research tech, prevents bricking
 	//NSV13 Added 3 ids for NSV13 hull materials
 	design_ids = list("basic_matter_bin", "basic_cell", "basic_scanning", "basic_capacitor", "basic_micro_laser", "micro_mani", "desttagger", "handlabel", "packagewrap",
-	"destructive_analyzer", "circuit_imprinter", "experimentor", "rdconsole", "design_disk", "tech_disk", "rdserver", "rdservercontrol", "mechfab", "paystand",
+	"destructive_analyzer", "circuit_imprinter", "experimentor", "rdconsole", "design_disk", "tech_disk", "rdserver", "gulagpack", "rdservercontrol", "mechfab", "paystand",
```

Comment:
I feel like this would be better under the "basic security equipment" node

Reply:
Done